src/screens/TopicPageScreen/MemoizedListComponent.js

```diff
@@ -148,6 +184,17 @@ const RenderListFeed = (props) => {
             setHaveSeeMore={(haveSeeMore) => setIsHaveSeeMore(haveSeeMore)}
           />
         )}
+        {isBlurred && (
+          <TopicsChip
+            onLayout={calculateLineTopicChip}
+            topics={item?.topics}
+            fontSize={normalizeFontSizeByWidth(14)}
+            text={item.message}
+            topicContainer={{
+              bottom: topicBottomPosition()
+            }}
+          />
+        )}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new `TopicsChip` component has been introduced which is conditionally rendered based on the `isBlurred` state. Make sure the `onLayout`, `topics`, `fontSize`, `text`, and `topicContainer` props are handled correctly in the `TopicsChip` component.
```javascript
{isBlurred && (
  <TopicsChip
    onLayout={calculateLineTopicChip}
    topics={item?.topics}
    fontSize={normalizeFontSizeByWidth(14)}
    text={item.message}
    topicContainer={{
      bottom: topicBottomPosition()
    }}
  />
)}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->